pkg/collector/stats/stats.go

```diff
@@ -75,7 +75,7 @@ func NewStats() *Stats {
 		m.ResourceUsage[metricName] = types.NewUInt64StatCollection()
 	}
 
-	if gpu.IsGPUCollectionSupported() {
+	if config.EnabledGPU {
```

Comment:
Why not use `gpu.IsGPUCollectionSupported()`?
A user could enable GPU in Kepler configuration but the system might not have GPUs.